tools/travis-install-protoc.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# ensure that the specified version of protoc is installed in
+# /tmp/proto$PROTO_VERSION/bin/protoc, which maybe cached
+
+# bash tools/travis-install-protoc.sh 3.0.0-beta-3.1
+#   
+
+
+# make bash more robust.
+set -beEux -o pipefail
```

Comment:
Haha agreed. I lifted it from the schemas. https://github.com/ga4gh/schemas/blob/master/tools/travis-install-protoc.sh Maybe @diekhans would like to explain this bash-fu.